ui/src/pages/search/topics/index.tsx

```diff
@@ -0,0 +1,323 @@
+                                name="pageSize"
+                                id="pageSize"
+                                onChange={handlePageSizeChange}
+                                value={limit}
+                                className="w-full rounded-md border border-grey-200 outline-none focus:ring-2 focus:ring-yellow-500"
+                                disabled={isValidating}
+                            >
+                                {pageSizes.map((pageSize) => (
+                                    <option key={pageSize} value={pageSize}>
+                                        {pageSize}
+                                    </option>
+                                ))}
+                            </select>
+                        </label>
+
+                        <form
+                            id="query-form"
+                            className="col-span-12 lg:col-span-7 xl:col-span-8"
+                            onSubmit={handleSearchFormSubmit}
+                        >
+                            <label htmlFor="searchTerm" className="relative block w-full">
+                                <span className="mb-1 block text-xxs font-bold uppercase tracking-widest text-grey-600 transition-colors duration-500 dark:text-grey-300">
+                                    Quick search
+                                </span>
+                                <input
+                                    name="searchTerm"
+                                    id="searchTerm"
+                                    defaultValue={query}
+                                    type="text"
+                                    placeholder="Type here and press enter..."
+                                    className="w-full rounded-md border border-grey-200 px-4 py-2 outline-none focus:ring-2 focus:ring-yellow-500 disabled:opacity-70"
+                                    disabled={isValidating}
+                                />
+                                <button
+                                    type="submit"
+                                    aria-label="Search"
+                                    className="absolute right-px rounded-md p-2 outline-none focus:ring-2 focus:ring-yellow-500 disabled:opacity-70"
+                                    disabled={isValidating}
+                                >
+                                    <SolidIcons.MagnifyingGlassIcon className="h-6 w-6 text-teal-500" />
+                                </button>
+                            </label>
+                        </form>
+                    </fieldset>
+
+                    <article className="col-span-12 min-h-screen">
+                        {props.error ? (
+                            <Components.Alert
+                                severity="ERROR"
+                                title={props.error}
+                                details={['Placeholder support text here']}
```

Comment:
I see we're using this placeholder text on the other search pages but shouldn't we just not pass this prop?

Reply:
Thanks -  it would make sense to also remove it from the publication + author search components now

Reply:
yep, done.